app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -409,6 +436,52 @@ export default {
         },
       });
     },
+    isEditorMouseFocusedOnAnImage() {
+      const selectedNode = document.querySelector(
+        'img.ProseMirror-selectednode'
+      );
+      if (selectedNode) {
+        this.selectedNode = selectedNode;
+        this.isImageSelected = !!selectedNode;
```

Comment:
```suggestion
      this.selectedNode = document.querySelector(
        'img.ProseMirror-selectednode'
      );
      if (this.selectedNode) {
        this.isImageSelected = true;
```

We can assign `this.selectedNode` directly, also `!!selectedNode` will always evaluate to true, right?

Reply:
> !!selectedNode will always evaluate to true, right?
Right